### PR TITLE
AAP-33750: lightspeed/role: add the role_name field

### PR DESCRIPTION
**Pull request by goneri**

### CODE REVIEW

File: src/features/lightspeed/roleGeneration.ts [hunk 1]
```diff
@@ -238,11 +238,12 @@ export async function showRoleGenerationPage(extensionUri: vscode.Uri) {
       }
       case "openEditor": {
         const { playbook } = message;
-        await openNewPlaybookEditor(playbook);
         await sendActionEvent(
           PlaybookGenerationActionType.CLOSE_ACCEPT,
           undefined,
         );
+        await openNewPlaybookEditor(playbook);
+        await openNewPlaybookEditor(playbook);
```
Comment (TamiTakamiya) [inline src/features/lightspeed/roleGeneration.ts:246]: Are these lines intentionally duplicated?

**Review by TamiTakamiya — COMMENTED** [review]
Left a question on a duplicated line.  I guess it's a typo.

(update) Actually it was NOT a typo.

File: src/features/lightspeed/roleGeneration.ts (hunk 1 above)
Comment (TamiTakamiya) [reply src/features/lightspeed/roleGeneration.ts:246]: I got an explanation from @goneri and understand that it was intentionally added. Resolving the conversation.

**Review by TamiTakamiya — APPROVED** [review]

**Review by ssbarnea — DISMISSED** [review]
Please add tests to avoid coverage going down.

**Comment by ssbarnea** [comment]
Marked it draft because is not passing CI.

**Pull request merged**